docs/source/users/index.md

```diff
@@ -205,6 +218,10 @@ Once you have started JupyterLab, click the new "chat" icon in the left side pan
     alt="Screen shot of the setup interface"
     class="screenshot" />
 
+:::{note}
+To activate the chat interface in Notebook 7, click on "View > Left sidebar > Show Jupyter AI Chat".
+:::
+
```

Comment:
What do you think about moving this section up to its first mention near line 50? I agree that it doesn't really fall under the "Prerequisites" section, but this information is critical for users who want to use Jupyter AI in Notebook 7. In my opinion, we should keep mentions of Notebook 7 local to one another if there are only a few mentions of it in this doc.

Reply:
Sure, sounds good!